### DISCUSSION TITLE
Missing Email Validation on Password Reset Page

### DISCUSSION THREAD

**Discussion by Ralkage** [requarks/wiki#6824] [Error / Bug Report]
Currently, the password reset page lacks validation for the email field, allowing users to proceed without entering a valid email address. Implementing proper validation will enhance security and user experience.

Tested on https://docs.requarks.io/login for reference.

https://github.com/requarks/wiki/assets/2059356/24eb870d-9c8a-4931-a0de-554f0c806b9d

**Comment by tim-elmer** [comment]
I just tested this accidentally by trying to determine why I wasn't getting password reset emails (with the incorrect address). Wiki.js doesn't actually send a reset email if the address is invalid, so it's possible this is intended to prevent revealing valid accounts.

That said, running a basic pattern match against the entered string to prevent user error (e.g. `\S@\S\`) might improve UX.
